src/components/FindWallet/WalletFilterPersona.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react"
+import {
+  Box,
+  Checkbox,
+  CheckboxGroup,
+  Flex,
+  Heading,
+  useColorModeValue,
+} from "@chakra-ui/react"
+
+import { trackCustomEvent } from "@/lib/utils/matomo"
+import { getPersonaBorderColor } from "@/lib/utils/wallets"
+
+import { useWalletPersonas } from "../../hooks/useWalletPersonas"
+
+const WalletFilterPersona = ({
+  resetFilters,
+  setFilters,
+  selectedPersona,
+  setSelectedPersona,
+}) => {
+  const personas = useWalletPersonas()
+  const filterBg = useColorModeValue("chakra-subtle-bg", "black400")
```

Comment:
Similar to the previous comment, I wonder where these colors come from since they are not defined in the DS. I'd like to see more reusability of the colors we've defined.

For reference, these are the [semantic tokens](https://github.com/ethereum/ethereum-org-website/blob/dev/src/@chakra-ui/semanticTokens.ts) we have.

cc @konopkja @nloureiro

Reply:
@pettinarip fixed